daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/MapKOps.scala

```diff
@@ -8,7 +8,8 @@ import scala.collection.GenTraversableOnce
 import scala.collection.immutable.{Map, MapLike}
 
 /** Halfway between the *-kinded MapLike and *->*->*-kinded MapOps. */
-trait MapKOps[K, +V, +This[+TV] <: Map[K, TV] with MapKOps[K, TV, This]] extends MapLike[K, V, This[V]] { this: This[V] =>
+trait MapKOps[K, +V, +This[+TV] <: Map[K, TV] with MapKOps[K, TV, This]]
+    extends MapLike[K, V, This[V]] { this: This[V] =>
```

Comment:
Seems unrelated (and there are a few others too). How come?

Reply:
don't know... `./fmt.sh` did this.